src/msg/simple/Accepter.cc

```diff
@@ -313,7 +324,21 @@ void *Accepter::entry()
       }
       ldout(msgr->cct,1) << __func__ << " poll got error"  
  			  << " errno " << errno << " " << cpp_strerror(errno) << dendl;
-      break;
+ 			  
+ 	  if (msgr->cct->get_module_type() == CEPH_ENTITY_TYPE_OSD && hb){
+ 	    if (!unhealthy){
+	      ldout(msgr->cct,0) << "pool got error"
+	      		<< " reset heartbeat timeout"<< dendl;
+          msgr->cct->get_heartbeat_map()->reset_timeout(hb,0,0);
+          unhealthy = true;
+        }
+        else{
```

Comment:
I know the messenger includes some of our oldest code, so maybe it's not obvious here. But our style guide dictates that the else block be on a single line, and that there be spaces between the word and the braces.
```} else {```

Also the spacing here looks very weird in Github. I'm not sure if that's a Github display issue or if the spacing in this patch doesn't follow our normal rules?